pkg/auth/providers/common/usermanager.go

```diff
@@ -311,26 +310,17 @@ func (m *userManager) EnsureClusterToken(clusterName string, input user.TokenInp
 
 // newTokenForKubeconfig creates a new token for a generated kubeconfig.
 func (m *userManager) newTokenForKubeconfig(clusterName, tokenName, description, kind, userName string, userPrincipal v3.Principal) (string, error) {
-	// settings.KubeconfigTokenTTLMinutes is deprecated use tokens.GetKubeconfigDefaultTokenTTLInMilliSeconds() when the setting is removed
-	tokenTTL, err := tokens.ParseTokenTTL(settings.KubeconfigTokenTTLMinutes.Get())
-	if err != nil {
-		return "", fmt.Errorf("failed to parse setting '%s': %w", settings.KubeconfigTokenTTLMinutes.Name, err)
-	}
 
-	tokenTTL, err = tokens.ClampToMaxTTL(tokenTTL)
```

Comment:
We can remove this reference to `ClampToMaxTTL` because `GetKubeconfigDefaultTokenTTLInMilliSeconds` already does this